### ISSUE TITLE
TypeError: Bad argument

### ISSUE THREAD

**Issue by rogervila** [imagemin/imagemin-jpegoptim#11]
Hello,

When I add jpegoptim in the <code>use</code> array, I get an error, but I can use the rest of the imagemin plugins.

This is the code:

<pre>
var gulp = require('gulp');

var imagemin = require('gulp-imagemin');
var pngquant = require('imagemin-pngquant');
var jpegoptim = require('imagemin-jpegoptim');
var jpegtran = require('imagemin-jpegtran');

gulp.task('optimg', () => {
    return gulp.src('public/assets/img/*')
        .pipe(imagemin({
            progressive: true,
            optimizationLevel: 4,
            svgoPlugins: [{removeViewBox: false}],
            use: [pngquant(),jpegoptim()] // <-- with jpegtran() there is no problem
        }))
        .pipe(gulp.dest('public/assets/img'));
});
</pre>

This is the error:

<pre>
internal/child_process.js:274
  var err = this._handle.spawn(options);
                         ^

TypeError: Bad argument
    at TypeError (native)
    at ChildProcess.spawn (internal/child_process.js:274:26)
    at exports.spawn (child_process.js:339:9)
    at Through2._transform (/usr/local/lib/node_modules/imagemin-jpegoptim/index.js:51:12)
    at Through2.Transform._read (/usr/local/lib/node_modules/imagemin-jpegoptim/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:184:10)
    at Through2.Transform._write (/usr/local/lib/node_modules/imagemin-jpegoptim/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:172:12)
    at doWrite (/usr/local/lib/node_modules/imagemin-jpegoptim/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:237:10)
    at writeOrBuffer (/usr/local/lib/node_modules/imagemin-jpegoptim/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:227:5)
    at Through2.Writable.write (/usr/local/lib/node_modules/imagemin-jpegoptim/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:194:11)
    at Through2.ondata (/usr/local/lib/node_modules/imagemin-pngquant/node_modules/readable-stream/lib/_stream_readable.js:581:20)
</pre>

Am I doing something wrong, or is it a bug?

Thank you very much.

**Comment by shinnn** [comment]
Can you show us the result of:

```
node --version
npm --version
npm ls imagemin imagemin-jpegoptim
```

And what is your OS?

**Comment by shinnn** [comment]
Never mind. Fixed in 4.1.0. https://github.com/imagemin/imagemin-jpegoptim/commit/4781cbee8d1b1e0af6d0d4853348da14260aa0fd

**Comment by rogervila** [comment]
Thank you very much @shinnn for your fast reply. 
I have updated the package but now I get another problem: the optimization works, but it reduces the images at 100%, so I get 0kb images as a result.
Should I change something on the code? I have tried with the <code>max</code> and <code>size</code> options but I get the same result.
![jpegoptim](https://cloud.githubusercontent.com/assets/6053012/11171589/561e37f6-8bf4-11e5-8204-5bc8cb9744a1.png)
If you want, I can open a new Issue with this problem.
Thank you very much
